config/initializers/devise.rb

```diff
@@ -264,12 +264,12 @@
 
   # Activate the test mode, useful when no real iChain is present, like in
   # testing and development environments
-  # config.ichain_test_mode = true
+  config.ichain_test_mode = true
 
```

Comment:
I uncommented these lines according to the instructions in the Contributing Guide for Using iChain in Test Mode. I will comment them out.